### PR TITLE
Cheat disable plant aging

### PR DESCRIPTION
**Pull request by Goddesen**
This PR adds a cheat for disabling plant aging, such that they don't wilt, as suggested in #3754

### CODE REVIEW

**Comment by JeroenDStout** [comment]
The only thing I am thinking about this is that (for plants) not wilting is the same as being watered regularly, but it puts the responsibility at the plant instead of the cheat that is called. I am thinking it might be beneficial to call any cheat regularly, rather than add more logic.

At the same time, that's exactly how fixing rides (button vs never break down) works, so perhaps this is consistent but could be expanded to vandalism and litter :)

**Comment by Nubbie** [comment]
I see this is your first PR, thanks for contributing :D 

Tested it out and it seems to work as intended for me :+1:

**Comment by Goddesen** [comment]
Would I be the one to add it, or is that up to a lead developer?

**Comment by Nubbie** [comment]
Think you could add it, if Im wrong, they will just say 'Noono' .o.

File: src/rct2/S6Exporter.cpp [hunk 1]
```diff
@@ -562,6 +562,7 @@ extern "C"
         SDL_WriteU8(rw, gCheatsDisableLittering);
         SDL_WriteU8(rw, gCheatsNeverendingMarketing);
         SDL_WriteU8(rw, gCheatsFreezeClimate);
+	SDL_WriteU8(rw, gCheatsDisablePlantAging);
```
Comment (HaasJona) [inline src/rct2/S6Exporter.cpp]: I think when changing these, you need to increment the network protocol version. (Also, the indentation doesn't match the lines above, they're using spaces)

**Comment by Gymnasiast** [comment]
Yes, you should add yourself under 'Additional implementation'.

**Comment by Goddesen** [comment]
Rebased and added feature to changelog.

File: src/rct2/S6Exporter.cpp [hunk 2]
```diff
@@ -562,6 +562,7 @@ extern "C"
         SDL_WriteU8(rw, gCheatsDisableLittering);
         SDL_WriteU8(rw, gCheatsNeverendingMarketing);
         SDL_WriteU8(rw, gCheatsFreezeClimate);
+        SDL_WriteU8(rw, gCheatsDisablePlantAging);
```
Comment (janisozaur) [inline src/rct2/S6Exporter.cpp]: Changes to this part require bumping [network version](https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/network/network.h#L58). There's another PR inbound also touching it, https://github.com/OpenRCT2/OpenRCT2/pull/3782, they can either go in together or there'll be minor conflict requiring rebase.

@IntelOrca any comments?

Comment (IntelOrca) [inline src/rct2/S6Exporter.cpp]: no comments

Comment (Goddesen) [inline src/rct2/S6Exporter.cpp]: Just leave it out for now, merge this first, include in #3782?

File: data/language/en-GB.txt [hunk 3]
```diff
@@ -4180,6 +4180,9 @@ STR_5868    :{WINDOW_COLOUR_2}Provider E-mail: {BLACK}{STRING}
 STR_5869    :{WINDOW_COLOUR_2}Provider Website: {BLACK}{STRING}
 STR_5870    :{SMALLFONT}{BLACK}Show server information
```
Comment (Gymnasiast) [inline data/language/en-GB.txt]: Remove this white line.

File: distribution/changelog.txt [hunk 4]
```diff
@@ -6,6 +6,7 @@
 - Feature: Allow setting ownership of map edges.
 - Feature: Allow up to 255 cars per train.
 - Feature: Importing SV4 and SC4 files with rides.
+- Feature: Add cheat: Disable plant aging.
```
Comment (Gymnasiast) [inline distribution/changelog.txt]: ```- Feature: Cheat to disable plant aging.```

**Comment by Gymnasiast** [comment]
I left two line notes. Other than that, the code is looking good. I still have to test it though.

**Comment by Goddesen** [comment]
Rebased to catch up with develop and edited to conform with comments from @Gymnasiast .

**Comment by janisozaur** [comment]
@Goddesen @Gymnasiast I will merge this PR to #3782 so we don't have to bump network version twice.

**Comment by Goddesen** [comment]
@janisozaur Please notice I removed the changes to the files in `rct2`. They will have to be added back in in a manner you see fit :)

**Comment by janisozaur** [comment]
@Goddesen it seemed fine, I will just skip 4f0c7bcefc22f8ce479ae8a6592f1502dd4337ea when merging

**Comment by janisozaur** [comment]
@Gymnasiast I understand from discussion on gitter you're OK with this, and I'm going to merge this into #3782 now.

**Pull request closed without merging**